types/catbox-memory/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "strict-export-declare-modifiers": false
```

Comment:
This disable also seems to be unnecessary?

Reply:
If I enable it my editor lights up like a christmas tree, I've even gone as far as to re-clone the entire DT project (you should really compress the history at some point) and it still does it.
I will remove it from my PR.

Reply:
@andy-ms 

![image](https://user-images.githubusercontent.com/3259089/48619757-d5bef900-e99d-11e8-8d92-74c386c78722.png)

Is this the expected behaviour?

Reply:
If it's not the behavior on the command line, no -- it sounds like an error in the editor plugin.